### PR TITLE
To fix the 'repo_states.json' file generating

### PR DESCRIPTION
**Pull request by asaechnx**
- this commit removes the error of the 'commit_id' field in the exported
  'repo_states.json' file

### CODE REVIEW

**Comment by azhogov** [comment]
Dima, please review too

File: common/git_worker.py [hunk 1]
```diff
@@ -238,6 +238,7 @@ def extract_all_repos(self):
 
         for repo in self.repo_states:
             if not repo.commit_id:
+                repo.commit_id = 'head'
```
Comment (dlobanox) [inline common/git_worker.py]: It's not quite obvious for me to understand why we set 'head' to commit_id here. 

It's better to update constructor of ProductState class and modify 'if' conditions in extract_all_repos method

Comment (asaechnx) [reply common/git_worker.py]: That is good idea, I'll try to test it.

File: common/git_worker.py [hunk 2]
```diff
@@ -213,7 +213,8 @@ def __init__(self, sources_list, root_repo_dir, commit_time):
 
         for repo_name, data in sources_list.items():
             branch = data.get('branch', 'master')
-            commit_id = data.get('commit_id')
+            commit_id = data.get('commit_id') if data.get('commit_id')
```
Comment (dlobanox) [inline common/git_worker.py]: commit_id = data.get('commit_id', 'HEAD')

231 line: if repo.commit_id != 'HEAD'
241 line: if repo.commit_id == 'HEAD'

**Pull request merged**